src/main/java/com/synopsys/integration/alert/web/model/NotificationContentConverter.java

```diff
@@ -49,8 +56,8 @@ public Config getConfigFromJson(final String json) {
     @Override
     public DatabaseEntity populateEntityFromConfig(final Config restModel) {
         final NotificationConfig notificationConfig = (NotificationConfig) restModel;
-        final Date createdAt = getContentConverter().getValue(notificationConfig.getCreatedAt(), Date.class);
```

Comment:
This way of converting String to Date throws exceptions. The method is apparently never called which is why we didn't catch it sooner